backend/tests/character/test_calc_vars.py

```diff
@@ -76,7 +76,7 @@ def test_multiple_raidbuff(self):
         the_shitters_i_raid_with = Comp(
```

Comment:
while you're cleaning up this file, can you change the variable name?

this was a dumb joke and i regret it now that this is a public repository

Reply:
fixed